app/views/api_docs/pages/release_notes.md

```diff
@@ -3,3 +3,8 @@
 Initial release of the API.
 
 For a log of pre-release changes, [see the alpha release notes](/api-docs/alpha-release-notes).
+
+#### 7th January 2020
+
+- Correct size of `personal_statement` field to accomodate 1000 words, not 600.
```

Comment:
Can we link to the properties? Would be good to add the new char limit as well (I don't think we mention "words" anywhere else?)

Reply:
Yes